.github/workflows/docs-review.yml

```
@@ -0,0 +1,29 @@
+# When a PR is labelled with 'docs-review-required',
+# this workflow comments on the PR to notify the GitHub CodeQL docs team.
+name: Request docs review
+on:
+  # Runs in the context of the base repo.
+  # This gives the workflow write access to comment on PRs.
+  # The workflow should not check out or build the given ref,
+  # or use untrusted data from the event payload in a command line.
+  pull_request_target:
+    types: [labeled]
+
+jobs:
+  request-docs-review:
+    name: Request docs review
+    # Run only on labelled PRs to the main repository.
+    # Do not run on PRs to forks.
+    if:
+      github.event.label.name == 'docs-review-required'
```

Maybe modify the label name slightly?

```suggestion
      github.event.label.name == 'docs-review-ready'
```

---

How about `ready-for-docs-review`, so it's clear that your review is being requested?

---

That sounds perfect 😄